src/services/resources/controllers.ts

```diff
@@ -14,6 +22,12 @@ const sendResponse = (expressRes: Response, logicResponse: ILogicResponse | IErr
         })
     }
 
+    if( "clearCookies" in logicResponse && logicResponse.clearCookies !== undefined ){
```

Comment:
I would over load the existing cookies interface rather than creating a separate one for clearing cookies

Reply:
going to use cookie.val === null to clear